docs/eventing/sink/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Sink"
+weight: 60
+type: "docs"
+---
+
+{{% readfile file="README.md" %}}
```

Comment:
I think we're not doing this with the readmes anymore, correct @RichieEscarez ?

Reply:
Tests were failing, so I re-added it, see https://github.com/knative/docs/pull/3031#issuecomment-726082942 :)

Reply:
@pierDipi Yes but there's no need to have an include/readme, the content should go directly into the index I think. Again @RichieEscarez or @evankanderson can probably confirm better.

Reply:
Ok.

Reply:
Done!